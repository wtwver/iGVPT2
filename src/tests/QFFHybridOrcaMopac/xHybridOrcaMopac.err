

          MOPAC Job: "/home/allouche/tmp/igvpt2_31679/h2oVQFF_0/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.037s
user	0m0.016s
sys	0m0.008s


          MOPAC Job: "/home/allouche/tmp/igvpt2_197/h2oVQFF_10/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.036s
user	0m0.016s
sys	0m0.008s


          MOPAC Job: "/home/allouche/tmp/igvpt2_32630/h2oVQFF_11/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.034s
user	0m0.016s
sys	0m0.008s


          MOPAC Job: "/home/allouche/tmp/igvpt2_20428/h2oVQFF_12/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.034s
user	0m0.020s
sys	0m0.012s


          MOPAC Job: "/home/allouche/tmp/igvpt2_5591/h2oVQFF_13/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.034s
user	0m0.016s
sys	0m0.008s


          MOPAC Job: "/home/allouche/tmp/igvpt2_5727/h2oVQFF_14/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.033s
user	0m0.016s
sys	0m0.004s


          MOPAC Job: "/home/allouche/tmp/igvpt2_9573/h2oVQFF_15/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.035s
user	0m0.016s
sys	0m0.008s


          MOPAC Job: "/home/allouche/tmp/igvpt2_25441/h2oVQFF_16/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.036s
user	0m0.020s
sys	0m0.004s


          MOPAC Job: "/home/allouche/tmp/igvpt2_3050/h2oVQFF_17/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.035s
user	0m0.020s
sys	0m0.004s


          MOPAC Job: "/home/allouche/tmp/igvpt2_27620/h2oVQFF_18/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.033s
user	0m0.012s
sys	0m0.012s


          MOPAC Job: "/home/allouche/tmp/igvpt2_1213/h2oVQFF_19/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.036s
user	0m0.024s
sys	0m0.000s


          MOPAC Job: "/home/allouche/tmp/igvpt2_22817/h2oVQFF_1/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.035s
user	0m0.020s
sys	0m0.004s


          MOPAC Job: "/home/allouche/tmp/igvpt2_25342/h2oVQFF_20/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.033s
user	0m0.016s
sys	0m0.004s


          MOPAC Job: "/home/allouche/tmp/igvpt2_18522/h2oVQFF_21/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.033s
user	0m0.016s
sys	0m0.004s


          MOPAC Job: "/home/allouche/tmp/igvpt2_11975/h2oVQFF_22/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.037s
user	0m0.020s
sys	0m0.004s


          MOPAC Job: "/home/allouche/tmp/igvpt2_18322/h2oVQFF_23/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.034s
user	0m0.024s
sys	0m0.000s


          MOPAC Job: "/home/allouche/tmp/igvpt2_26786/h2oVQFF_24/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.031s
user	0m0.016s
sys	0m0.000s


          MOPAC Job: "/home/allouche/tmp/igvpt2_15265/h2oVQFF_25/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.030s
user	0m0.016s
sys	0m0.004s


          MOPAC Job: "/home/allouche/tmp/igvpt2_27397/h2oVQFF_26/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.033s
user	0m0.016s
sys	0m0.004s


          MOPAC Job: "/home/allouche/tmp/igvpt2_740/h2oVQFF_27/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.035s
user	0m0.016s
sys	0m0.008s


          MOPAC Job: "/home/allouche/tmp/igvpt2_13817/h2oVQFF_28/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.034s
user	0m0.020s
sys	0m0.004s


          MOPAC Job: "/home/allouche/tmp/igvpt2_25381/h2oVQFF_29/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.035s
user	0m0.024s
sys	0m0.000s


          MOPAC Job: "/home/allouche/tmp/igvpt2_30341/h2oVQFF_2/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.035s
user	0m0.024s
sys	0m0.000s


          MOPAC Job: "/home/allouche/tmp/igvpt2_27337/h2oVQFF_30/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.033s
user	0m0.012s
sys	0m0.012s


          MOPAC Job: "/home/allouche/tmp/igvpt2_6057/h2oVQFF_31/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.034s
user	0m0.012s
sys	0m0.012s


          MOPAC Job: "/home/allouche/tmp/igvpt2_2997/h2oVQFF_32/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.034s
user	0m0.020s
sys	0m0.000s


          MOPAC Job: "/home/allouche/tmp/igvpt2_11225/h2oVQFF_33/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.035s
user	0m0.020s
sys	0m0.000s


          MOPAC Job: "/home/allouche/tmp/igvpt2_9309/h2oVQFF_34/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.035s
user	0m0.020s
sys	0m0.004s


          MOPAC Job: "/home/allouche/tmp/igvpt2_1953/h2oVQFF_35/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.034s
user	0m0.020s
sys	0m0.004s


          MOPAC Job: "/home/allouche/tmp/igvpt2_18748/h2oVQFF_36/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.037s
user	0m0.016s
sys	0m0.008s


          MOPAC Job: "/home/allouche/tmp/igvpt2_16877/h2oVQFF_37/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.032s
user	0m0.012s
sys	0m0.008s


          MOPAC Job: "/home/allouche/tmp/igvpt2_21145/h2oVQFF_38/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.037s
user	0m0.020s
sys	0m0.004s


          MOPAC Job: "/home/allouche/tmp/igvpt2_8375/h2oVQFF_39/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.033s
user	0m0.020s
sys	0m0.004s


          MOPAC Job: "/home/allouche/tmp/igvpt2_10310/h2oVQFF_3/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.032s
user	0m0.020s
sys	0m0.000s


          MOPAC Job: "/home/allouche/tmp/igvpt2_26153/h2oVQFF_40/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.036s
user	0m0.016s
sys	0m0.008s


          MOPAC Job: "/home/allouche/tmp/igvpt2_5711/h2oVQFF_41/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.034s
user	0m0.020s
sys	0m0.004s


          MOPAC Job: "/home/allouche/tmp/igvpt2_9103/h2oVQFF_42/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.032s
user	0m0.016s
sys	0m0.004s


          MOPAC Job: "/home/allouche/tmp/igvpt2_860/h2oVQFF_43/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.032s
user	0m0.016s
sys	0m0.004s
mkdir: impossible de créer le répertoire « /home/allouche/tmp/igvpt2_10535 »: Le fichier existe


          MOPAC Job: "/home/allouche/tmp/igvpt2_10535/h2oVQFF_44/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.035s
user	0m0.020s
sys	0m0.004s
mkdir: impossible de créer le répertoire « /home/allouche/tmp/igvpt2_12619 »: Le fichier existe


          MOPAC Job: "/home/allouche/tmp/igvpt2_12619/h2oVQFF_45/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.033s
user	0m0.020s
sys	0m0.004s


          MOPAC Job: "/home/allouche/tmp/igvpt2_30045/h2oVQFF_46/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.035s
user	0m0.016s
sys	0m0.004s


          MOPAC Job: "/home/allouche/tmp/igvpt2_10746/h2oVQFF_47/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.035s
user	0m0.016s
sys	0m0.008s


          MOPAC Job: "/home/allouche/tmp/igvpt2_27529/h2oVQFF_48/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.035s
user	0m0.020s
sys	0m0.000s


          MOPAC Job: "/home/allouche/tmp/igvpt2_31356/h2oVQFF_49/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.034s
user	0m0.016s
sys	0m0.008s


          MOPAC Job: "/home/allouche/tmp/igvpt2_28670/h2oVQFF_4/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.031s
user	0m0.016s
sys	0m0.004s


          MOPAC Job: "/home/allouche/tmp/igvpt2_23882/h2oVQFF_50/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.031s
user	0m0.016s
sys	0m0.000s


          MOPAC Job: "/home/allouche/tmp/igvpt2_118/h2oVQFF_51/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.034s
user	0m0.020s
sys	0m0.004s


          MOPAC Job: "/home/allouche/tmp/igvpt2_27525/h2oVQFF_52/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.037s
user	0m0.016s
sys	0m0.008s


          MOPAC Job: "/home/allouche/tmp/igvpt2_7930/h2oVQFF_53/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.034s
user	0m0.020s
sys	0m0.004s


          MOPAC Job: "/home/allouche/tmp/igvpt2_10568/h2oVQFF_54/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.033s
user	0m0.020s
sys	0m0.000s


          MOPAC Job: "/home/allouche/tmp/igvpt2_9614/h2oVQFF_5/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.034s
user	0m0.016s
sys	0m0.004s


          MOPAC Job: "/home/allouche/tmp/igvpt2_4960/h2oVQFF_6/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.035s
user	0m0.012s
sys	0m0.012s


          MOPAC Job: "/home/allouche/tmp/igvpt2_22948/h2oVQFF_7/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.033s
user	0m0.020s
sys	0m0.000s


          MOPAC Job: "/home/allouche/tmp/igvpt2_27262/h2oVQFF_8/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.035s
user	0m0.020s
sys	0m0.000s


          MOPAC Job: "/home/allouche/tmp/igvpt2_3056/h2oVQFF_9/One0.mop" ended normally on Jul 10, 2016, at 18:34.


real	0m0.036s
user	0m0.020s
sys	0m0.004s
