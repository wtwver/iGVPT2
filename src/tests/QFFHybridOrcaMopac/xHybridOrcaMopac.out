seed = -1
----------------------------------------------------------
runType=VPT2
model=MOPAC
----------------------------------------------------------

nFrequencies=3
VPT2 model : GVPT2

Gradients
END

Hessian
1 1      1642.7927630000
2 2      3833.9971290000
3 3      3943.9539900000
END

Cubic
1 1 1      -129.2170060000
1 1 2       542.0363960000
1 1 3         1.5109420000
2 2 1      -177.5916280000
2 2 2     -1244.4529740000
2 2 3         0.0559610000
3 3 1      -181.1731190000
3 3 2     -1337.7991010000
3 3 3         0.3847310000
END

Quartic
1 1 1 1      -189.4366650000
1 1 1 2       112.3203390000
1 1 1 3        -0.1049270000
2 2 1 1      -437.6135270000
2 2 2 1        63.4245820000
2 2 2 2        12.8989660000
2 2 2 3        -0.1503950000
3 3 1 1      -296.2565230000
3 3 2 2       138.1112770000
3 3 3 1         0.0664530000
3 3 3 2        -0.0069950000
3 3 3 3       135.9288050000
END

Coriolis
END

Rotation constants
1         1.0000000000
2         1.0000000000
3         1.0000000000
END
nFrequencies=3
nDim=3
VPT2 Properties model : GVPT2 : Gaussian method

First derivatives
1 1        -1.1216980489
1 2        -0.4191720674
1 3        -0.8550307868
2 1        -1.2895658753
2 2        -0.4565350054
2 3         2.5344421287
3 1        -2.6153519277
3 2        -0.9036898828
3 3        -0.8849731549
END


Second derivatives
1 1 1        -3.2174690000
1 1 2         0.9874460000
1 1 3         2.0030110000
1 2 1         0.9874460000
1 2 2         6.8050520000
1 2 3        -4.5899630000
1 3 1         2.0030110000
1 3 2        -4.5899630000
1 3 3         0.8620960000
2 1 1        -2.8871040000
2 1 2         2.9623380000
2 1 3        -6.0090340000
2 2 1         2.9623380000
2 2 2         6.8050520000
2 2 3         9.1799260000
2 3 1        -6.0090340000
2 3 2         9.1799260000
2 3 3         0.7241600000
3 1 1        -2.6141940000
3 1 2         3.9497840000
3 1 3         2.0030110000
3 2 1         3.9497840000
3 2 2         4.6260940000
3 2 3        -1.5299880000
3 3 1         2.0030110000
3 3 2        -1.5299880000
3 3 3         0.7241600000
END


Cubic derivatives
1 1 1 1      -235.7831660000
1 1 1 2      -160.0902120000
1 1 2 1      -160.0902120000
1 1 2 2      -244.5678020000
1 2 1 1      -160.0902120000
1 2 1 2      -244.5678020000
1 2 2 1      -244.5678020000
1 2 2 2      -560.4347290000
1 2 2 3       378.9429310000
1 2 3 2       378.9429310000
1 2 3 3      -384.3384530000
1 3 2 2       378.9429310000
1 3 2 3      -384.3384530000
1 3 3 2      -384.3384530000
1 3 3 3       682.1688980000
2 1 1 1      -392.9719440000
2 1 1 2      -160.0902120000
2 1 1 3       324.7392640000
2 1 2 1      -160.0902120000
2 1 2 2      -244.5678020000
2 1 3 1       324.7392640000
2 2 1 1      -160.0902120000
2 2 1 2      -244.5678020000
2 2 2 1      -244.5678020000
2 2 2 2      -560.4347290000
2 3 1 1       324.7392640000
2 3 3 3       682.1688980000
3 1 1 1      -392.9719440000
3 2 2 2       186.8115760000
3 2 2 3       378.9429310000
3 2 3 2       378.9429310000
3 2 3 3      -384.3384530000
3 3 2 2       378.9429310000
3 3 2 3      -384.3384530000
3 3 3 2      -384.3384530000
3 3 3 3       682.1688980000
END

nFrequencies 3
nStates 10
State n 1	0 0 0 
State n 2	1 0 0 
State n 3	0 1 0 
State n 4	0 0 1 
State n 5	2 0 0 
State n 6	0 2 0 
State n 7	0 0 2 
State n 8	1 1 0 
State n 9	1 0 1 
State n 10	0 1 1 

Xkl (cm^-1) by VPT2+K model
1 1       -40.5089214194
1 2        -8.1807278679
1 3       -31.4306124920
2 1        -8.1807278679
2 2       -43.6118291004
2 3      -153.1951172658
3 1       -31.4306124920
3 2      -153.1951172658
3 3       -43.2703979861
END

C=-2.924537

Diagonal of H (cm^-1) by VPT2+K model
     4630.3225394671         0.0000000000 
     6172.2917894482      1541.9692499811  1(1) 
     8296.4080876993      3666.0855482323  2(1) 
     8395.4228686159      3765.1003291489  3(1) 
     7633.2431965904      3002.9206571233  1(2) 
    11875.2699777307      7244.9474382636  2(2) 
    12073.9824017926      7443.6598623255  3(2) 
     9830.1966098125      5199.8740703455  1(1)  2(1) 
     9905.9615061050      5275.6389666379  1(1)  3(1) 
    11908.3132995824      7277.9907601153  2(1)  3(1) 
END


Resonance Matrix (cm^-1) by VPT2+K model
========================================
 Fermi Resonances terms :
 0 Fermi Resonances 
----------------------------------------------------------------------------------------------------

 Darling-Dennison Resonances terms :
 0 Darling-Dennison Resonances 
----------------------------------------------------------------------------------------------------

END



Results
========================
       E(harm)      E(anharm)     Mode(Quanta), Coef 
       -------      ---------     ---------------- 
    0.00000000     0.00000000     [ 1.000] 
 1642.79276300  1541.96924998     [ 1.000  1(1)] 
 3285.58552600  3002.92065712     [ 1.000  1(2)] 
 3833.99712900  3666.08554823     [ 1.000  2(1)] 
 3943.95399000  3765.10032915     [ 1.000  3(1)] 
 5476.78989200  5199.87407035     [ 1.000  1(1)  2(1)] 
 5586.74675300  5275.63896664     [ 1.000  1(1)  3(1)] 
 7667.99425800  7244.94743826     [ 1.000  2(2)] 
 7777.95111900  7277.99076012     [ 1.000  2(1)  3(1)] 
 7887.90798000  7443.65986233     [ 1.000  3(2)] 



Results With intensities
========================
       E(harm)      E(anharm)        I(harm)      I(anharm)     Coef Mode(Quanta) 
       -------      ---------        -------      ---------     ---------------- 
    0.00000000     0.00000000     0.00000000     0.00000000     [ 1.000] 
 1642.79276300  1541.96924998    79.03737739    83.05743175     [ 1.000  1(1)] 
 3285.58552600  3002.92065712     0.00000000     0.46561291     [ 1.000  1(2)] 
 3833.99712900  3666.08554823     9.72281843    10.34689428     [ 1.000  2(1)] 
 3943.95399000  3765.10032915    64.27158893    66.24116882     [ 1.000  3(1)] 
 5476.78989200  5199.87407035     0.00000000     0.00587024     [ 1.000  1(1)  2(1)] 
 5586.74675300  5275.63896664     0.00000000     0.46325942     [ 1.000  1(1)  3(1)] 
 7667.99425800  7244.94743826     0.00000000     0.35592107     [ 1.000  2(2)] 
 7777.95111900  7277.99076012     0.00000000     0.21850620     [ 1.000  2(1)  3(1)] 
 7887.90798000  7443.65986233     0.00000000     0.10264844     [ 1.000  3(2)] 



Results With intensities
========================

Fundamental Bands
-----------------
       E(harm)      E(anharm)        I(harm)      I(anharm)       Coef. Mode(Quanta) 
  ------------   ------------   ------------   ------------     -------------------- 
 1642.79276300  1541.96924998    79.03737739    83.05743175     [ 1.000  1(1)] 
 3833.99712900  3666.08554823     9.72281843    10.34689428     [ 1.000  2(1)] 
 3943.95399000  3765.10032915    64.27158893    66.24116882     [ 1.000  3(1)] 

Overtones
---------
       E(harm)      E(anharm)        I(harm)      I(anharm)       Coef. Mode(Quanta) 
  ------------   ------------   ------------   ------------     -------------------- 
 3285.58552600  3002.92065712     0.00000000     0.46561291     [ 1.000  1(2)] 
 7667.99425800  7244.94743826     0.00000000     0.35592107     [ 1.000  2(2)] 
 7887.90798000  7443.65986233     0.00000000     0.10264844     [ 1.000  3(2)] 

Combination Bands
----------------
       E(harm)      E(anharm)        I(harm)      I(anharm)       Coef. Mode(Quanta) 
  ------------   ------------   ------------   ------------     -------------------- 
 5476.78989200  5199.87407035     0.00000000     0.00587024     [ 1.000  1(1)  2(1)] 
 5586.74675300  5275.63896664     0.00000000     0.46325942     [ 1.000  1(1)  3(1)] 
 7777.95111900  7277.99076012     0.00000000     0.21850620     [ 1.000  2(1)  3(1)] 


